src/pages/profilePage.jsx

```diff
@@ -1,145 +1,180 @@
+  updatePhotoUrl
+} from "../hooks/databaseHooks";
 
 function Profile(props) {
-
-    const allInputs = {
-        imgUrl: ''
-    };
-    const [imageAsFile, setImageAsFile] = useState('');
-    const [tempName, setTempName] = useState(props.displayName);
-    const [imageAsURL, setImageAsUrl] = useState(allInputs);
-    const [profileImageURL, setProfileImageURL] = useState('');
-
-
-    /*Code adapted from Tallan Groberg
+  const allInputs = {
+    imgUrl: ""
+  };
+  const [imageAsFile, setImageAsFile] = useState("");
+  const [tempName, setTempName] = useState(props.displayName);
+  const [imageAsURL, setImageAsUrl] = useState(allInputs);
+  const [profileImageURL, setProfileImageURL] = useState("");
+  const [isEnabled, setEnabled] = useState(false);
+
+  /*Code adapted from Tallan Groberg
     here is the link to his tutorial https://dev.to/tallangroberg/how-to-do-image-upload-with-firebase-in-react-cpj*/
 
-    const handleImageAsFile = (e) => {
-        const image = e.target.files[0]
-        setImageAsFile(imageFile => (image))
-    }
-
-    async function handleTempName(newVal) {
-        setTempName(newVal.target.value);
-        await updateDisplayName(newVal.target.value);
-    }
-
-    function handleNameSubmit(newVal){
-        props.handleName(newVal.target.value);
-    }
-
-    const handleFireBaseUpload = e => {
-        let firebase = require("firebase/app");
-        let user = firebase.auth().currentUser;
-        e.preventDefault()
-        console.log('start of upload')
-        // async magic goes here...
-        if (imageAsFile === '') {
-            console.error(`not an image, the image file is a ${
-                typeof(imageAsFile)
-            }`)
-        }
-        const uploadTask = storage.ref(`/images/${
-            user.email.substring(0, user.email.lastIndexOf("@"))
-        }`).put(imageAsFile)
-        // initiates the firebase side uploading
-        uploadTask.on('state_changed', (snapShot) => { // takes a snap shot of the process as it is happening
-            console.log(snapShot)
-        }, (err) => { // catches the errors
-            console.log(err)
-        }, () => {
-            // gets the functions from storage refences the image storage in firebase by the children
-            // gets the download url then sets the image from firebase as the value for the imgUrl key:
-            firebase.auth().onAuthStateChanged(function(user) {
-                var mySubString = user.email.substring(
-                    0, 
-                    user.email.lastIndexOf("@")
-                )+ "_200x200";
-            storage.ref('images').child(mySubString).getDownloadURL().then(fireBaseUrl => {
-                setImageAsUrl(prevObject => ({
-                    ...prevObject,
-                    imgUrl: fireBaseUrl
-                }))
-            })});
-        })
+  const handleImageAsFile = e => {
+    const image = e.target.files[0];
+    setImageAsFile(imageFile => image);
```

Comment:
Also, if you are going for a dict, wouldn't you want this to be "imgUrl => image"?

Reply:
Made these changes